.github/workflows/pr.yml

```
@@ -27,14 +24,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

goreleaser needs this:
```suggestion
        uses: actions/checkout@v3
        with:
          fetch-depth: 0
```
Mostly to be able to have tags to tag the release that we are forcing with `GORELEASER_CURRENT_TAG=${VERSION}` but I would prefer to have it just in case.